backend/src/main/java/wooteco/prolog/studylog/ui/ReportController.java

```diff
@@ -0,0 +1,190 @@
+package wooteco.prolog.studylog.ui;
+
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.prolog.login.domain.AuthMemberPrincipal;
+import wooteco.prolog.member.domain.Member;
+
+@RestController
+@RequestMapping("/reports")
+public class ReportController {
+
+    @GetMapping("/{memberName}/{reportId}")
```

Comment:
url의 resource hierarchy가 조금 어색한 느낌이 듭니다.
report의 Id가 사람마다 1부터 시작하는게 아니라면 membername이 빠져도 좋을 것 같아요

```suggestion
    @GetMapping("/{reportId}")
```